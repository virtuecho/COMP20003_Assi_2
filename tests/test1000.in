Summerholm
Pinevale
Carstairs
Mount Eliza
Carlton
Waranga Shores
Hall
Tooloon
Rocky Point
Owen
Lower Mount Walker
Bohena Creek
Roxby Downs
Horsnell Gully
Faulconbridge
Kangaroo Point
Kamarooka
Wonderland
Stratheden
Spotswood
Tenterden
Flinders
Beeron
Springfield
Corop
Electra
Moolerr
Lyneham
Far Far Away
Nashua
Premaydena
Shoal Bay
Camden Head
Wagin
Kensington
Springfield
Hopeland
Dee
Boosey
Wheeo
Pasadena
Kevington
Kuraby
Deepdene
Nelligen
Mount Macarthur
Algorithms & Data Structure
Merlwood
Myrtle Mountain
Port Julia
Wirrimbi
Donald Creek
Tenterfield
Commissioners Creek
Manangatang
Nearum
New York
Blackbutt
Gosforth
Coffs Harbour
Mount Mort
Reedy Dam
Never Ending
Wybung
Gobarralong
Richmond Lowlands
Nyarrin
Evanslea
Balliang
Fadden
Lynton
Hoddys Well
Seddon
Bungundarra
Woodpark
Recherche
Carlton River
Wolvi
Binary Search Tree
Pyengana
Mount Barker Junction
Littlehampton
Gilston
Cuprona
Broadwater
Lisle
Ascot Vale
Wilsons Pocket
Argyll
Richmond
Port Bonython
