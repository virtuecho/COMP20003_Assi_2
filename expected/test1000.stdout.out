Summerholm --> 1 records found
Pinevale --> 1 records found
Carstairs --> 1 records found
Mount Eliza --> 1 records found
Carlton --> 3 records found
Waranga Shores --> 1 records found
Hall --> 1 records found
Tooloon --> 1 records found
Rocky Point --> 2 records found
Owen --> 1 records found
Lower Mount Walker --> 1 records found
Bohena Creek --> 1 records found
Roxby Downs --> 1 records found
Horsnell Gully --> 1 records found
Faulconbridge --> 1 records found
Kangaroo Point --> 1 records found
Kamarooka --> 1 records found
Wonderland --> NOT FOUND
Stratheden --> 1 records found
Spotswood --> 1 records found
Tenterden --> 1 records found
Flinders --> 1 records found
Beeron --> 1 records found
Springfield --> 9 records found
Corop --> 1 records found
Electra --> 1 records found
Moolerr --> 1 records found
Lyneham --> 1 records found
Far Far Away --> NOT FOUND
Nashua --> 1 records found
Premaydena --> 1 records found
Shoal Bay --> 1 records found
Camden Head --> 1 records found
Wagin --> 1 records found
Kensington --> 5 records found
Springfield --> 9 records found
Hopeland --> 1 records found
Dee --> 1 records found
Boosey --> 1 records found
Wheeo --> 1 records found
Pasadena --> 1 records found
Kevington --> 1 records found
Kuraby --> 1 records found
Deepdene --> 1 records found
Nelligen --> 1 records found
Mount Macarthur --> 1 records found
Algorithms & Data Structure --> NOT FOUND
Merlwood --> 1 records found
Myrtle Mountain --> 1 records found
Port Julia --> 1 records found
Wirrimbi --> 1 records found
Donald Creek --> 1 records found
Tenterfield --> 1 records found
Commissioners Creek --> 1 records found
Manangatang --> 1 records found
Nearum --> 1 records found
New York --> NOT FOUND
Blackbutt --> 1 records found
Gosforth --> 1 records found
Coffs Harbour --> 1 records found
Mount Mort --> 1 records found
Reedy Dam --> 1 records found
Never Ending --> NOT FOUND
Wybung --> 1 records found
Gobarralong --> 1 records found
Richmond Lowlands --> 1 records found
Nyarrin --> 1 records found
Evanslea --> 1 records found
Balliang --> 1 records found
Fadden --> 1 records found
Lynton --> 1 records found
Hoddys Well --> 1 records found
Seddon --> 2 records found
Bungundarra --> 1 records found
Woodpark --> 1 records found
Recherche --> 1 records found
Carlton River --> 1 records found
Wolvi --> 1 records found
Binary Search Tree --> NOT FOUND
Pyengana --> 1 records found
Mount Barker Junction --> 1 records found
Littlehampton --> 1 records found
Gilston --> 1 records found
Cuprona --> 1 records found
Broadwater --> 1 records found
Lisle --> 1 records found
Ascot Vale --> 1 records found
Wilsons Pocket --> 1 records found
Argyll --> 1 records found
Richmond --> 6 records found
Port Bonython --> 1 records found
